scripts/translate_schemas.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+import sys
+import subprocess
+
+import docker
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+TRANSLATION_MAP = {'cy': ['census_individual_gb_wls']}
+
+
+def translate_schemas():
+    docker_client = docker.from_env()
+
+    for language, schemas in TRANSLATION_MAP.items():
+        for schema_name in schemas:
+            translation_file = f'{schema_name}_{language}.po'
+            schema_file = f'{schema_name}.json'
+            relative_dir = f'data/{language}'
+            output_dir = f'/usr/src/eq-survey-runner/data/{language}'
+            language_dir = f'/usr/src/eq-survey-runner/app/translations/{language}'
+
+            os.makedirs(relative_dir, exist_ok=True)
+
+            docker_client.images.pull('onsdigital/eq-translations:latest')
+            container = docker_client.containers.run(
+                'onsdigital/eq-translations',
+                f'pipenv run python -m cli.translate_survey ../eq-survey-runner/data/en/{schema_file} {language_dir}/{translation_file} {output_dir}',
+                volumes={
+                    os.getcwd(): {'bind': '/usr/src/eq-survey-runner', 'mode': 'rw'}
+                },
+                remove=True,
+                detach=True,
+            )
+            for log in container.logs(stdout=True, stream=True):
+                logger.info(log.decode().strip('\n'))
+            logger.info('Building %s/%s', output_dir, translation_file)
+
+
+if __name__ == '__main__':
+    subprocess.run(['pipenv', 'run', 'pybabel', 'compile', '-d', 'app/translations'])
```

Comment:
Maybe the name of the script needs to be changed instead?

Reply:
I kind of think we need one command to "do translations" rather than needing to know that there are different types of translations across multiple commands..

Reply:
That's what the Makefile provides..

Reply:
Ah - I see, originally misread that.